main.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pandas as pd
+import dxpy as dx
+import datetime
+
+from util import read_yaml, dx_login, send_mail
+
+
+# define number of seqs
+seq = read_yaml('seq.yml')
+
+
+def main():
+
+    dx_login()
+
+    # defining directories in docker container
+    # Directories need to be mounted with docker run -v
+    GENETIC_DIR = 'var/genetics'
```

Comment:
can these be in the config file in case we change them so it doesn't require changing the code for updating paths

Reply:
got it!